### DISCUSSION TITLE
Cannot create neighborgoods in MoBi when expanding the included PBPK model

### DISCUSSION THREAD

**Discussion by ebanvil** [Open-Systems-Pharmacology/Forum#1287] [Questions & Problems] [answered]
Hello all,

I'm trying to expand the included PBPK model in PK-Sim **to include a new organ (eye) using MoBi**. 
I'm following the instructions described in https://ascpt.onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1002%2Fpsp4.12300&file=psp412300-sup-0001-suppinfo01.pdf and in this discussion (https://github.com/Open-Systems-Pharmacology/Forum/discussions/1180) but I'm **having trouble creating the neighborhoods** for the new organ.

This is what I see in the diagram:
![image](https://user-images.githubusercontent.com/46683252/186895185-2efde3df-c778-477d-9f3a-3ffcb17c3e8b.png)

But when I expand the new container, all the other containers dissapear, so I cannot create the connections.
![image](https://user-images.githubusercontent.com/46683252/186895418-9bc84860-9b55-467f-aab5-2524c7048aff.png)

I'm new using MoBi and this issue probably has an easy answer, but I've tried all the Layout options and searched the forum and tutorials and still could not solve the problem.

Thank you in advance,
Enrique

**Comment by wilbertdew** [comment, marked as answer]
Hi Enrique, two options to avoid this: 

1. If the 'Shift' key is pressed when clicking on the “+” sign, the other compartments will not disappear.
2. If the other containers are not visible, right-click anywhere in the empty space of the model structure and select 'Show All Children'.

**Reply by ebanvil** [reply]
Thank you very much wilbertdew!
I tried both options and they work. I also found another one: right click -> Expert Functions -> Expand / Collapse -> Expand Selection. In any case, I think the first option is the easiest one.
